src/cli/core/GeneratedFileWriter.ts

```diff
@@ -128,10 +154,11 @@ export class GeneratedFileWriter {
         })
         .catch(error => {
           this.printCliMessage('error', `Unable to read the file with custom types. ${error}`);
+          throw new Error();
         });
-
-      throw new Error();
```

Comment:
This section was always throwing a silent error. I've changed it to only emit an error after logging to the console.

Reply:
Good catch :+1: